packages/emotion-primitives/src/props.js

```diff
@@ -0,0 +1,18 @@
+import viewStylePropTypes from 'react-primitives/lib/web/View/ViewStylePropTypes'
+import textStylePropTypes from 'react-primitives/lib/web/Text/TextStylePropTypes'
+import imageStylePropTypes from 'react-primitives/lib/web/Image/ImageStylePropTypes'
+
+export const isValidStyleProp = (element, propName) => {
```

Comment:
similar history here, there is no need to cast those to array of strings with `Object.keys` and check later existance with `indexOf`

we can just do a simple lookup here